docker/docker-init.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
`bash` instead of `sh` in order to use `[[ =~ ]]` string matching later in the file.